motr/obj.c

```diff
@@ -555,6 +555,30 @@ static int obj_namei_op_init(struct m0_entity *entity,
 	return M0_ERR(rc);
 }
 
+/**
+ * Finds and sets optimal layout id according to the object's size.
+ */
+static void obj_optimal_lid_set(struct m0_obj *obj,
+				struct m0_layout_domain *ldom)
```

Comment:
I think the "ldom" can be declared as "const".

Also, please add comments to say that:
"obj->ob_attr.oa_layout_id will be updated with optimal layout id".

Reply:
Done